pages/03-core/no-data-states.page.tsx

```diff
@@ -0,0 +1,125 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import Highcharts from "highcharts";
+import { omit } from "lodash";
+
+import Box from "@cloudscape-design/components/box";
+
+import CoreChart from "../../lib/components/internal-do-not-use/core-chart";
+import { PageSettings, PageSettingsForm, useChartSettings } from "../common/page-settings";
+import { FramedDemo, Page, PageSection } from "../common/templates";
+
+interface ThisPageSettings extends PageSettings {
+  showInLegend?: boolean;
+}
+
+export default function () {
+  const { chartProps } = useChartSettings<ThisPageSettings>();
+  return (
+    <Page
+      title="Core chart: no data states"
+      subtitle="The page demonstrates all possible no-data states of the core chart."
+      settings={<PageSettingsForm selectedSettings={["height", "showLegend"]} />}
+    >
+      <PageSection title="Empty state: all" subtitle="No series provided">
+        <FramedDemo>
+          <CoreChart
+            {...omit(chartProps.cartesian, "ref")}
+            highcharts={Highcharts}
+            options={{
+              series: [],
+            }}
+            // legend={{ ...chartProps.cartesian.legend, position: "side" }}
+            noData={{
+              statusType: "finished",
+              empty: (
+                <>
+                  <Box textAlign="center" color="inherit">
+                    <b>No data available</b>
+                    <Box variant="p" color="inherit">
+                      There is no available data to display
+                    </Box>
+                  </Box>
+                </>
+              ),
+            }}
+          />
+        </FramedDemo>
+      </PageSection>
+
+      <PageSection title="Empty state: all (side legend)" subtitle="No series provided">
+        <FramedDemo>
+          <CoreChart
+            {...omit(chartProps.cartesian, "ref")}
+            highcharts={Highcharts}
+            options={{
+              series: [],
+            }}
+            legend={{ ...chartProps.cartesian.legend, position: "side" }}
+            noData={{
+              statusType: "finished",
+              empty: (
+                <>
```

Comment:
I would reuse the empty state across charts. As you use the same config, you can create a var `noData` and pass it to every chart